lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -230,17 +230,30 @@ def __init__(self, ufeSceneItem):
         # Get the UFE Attributes interface for this scene item.
         self.attrS = ufe.Attributes.attributes(self.item)
 
+        self.showArrayAttributes = cmds.optionVar(query="mayaUSD_AEShowArrayAttributes")
```

Comment:
There is no default value for this optionVar, so if the user never opens the show menu it will not be set. Therefore you should account for that here and first set your class var to a good default (false), then test if the optionVar exists and if so query it.